Rectangle/mul.lproj/Main.xcstrings

```diff
@@ -13281,7 +13461,7 @@
         "vi" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Bỏ qua frontmost.app"
+            "value" : "Bỏ qua ứng dụng trước nhất"
```

Comment:
frontmost.app will get programmatically replaced by the name of the frontmost app, so there is no need to translate this.